src/routing-lib/src/routing_lib.egg-info/dependency_links.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
Why empty file?

Reply:
The entire `routing-lib` project is just Ludovico's [own routing library](https://github.com/lwdovico/routing-lib) that we've downloaded in order to change dependencies, so I can't really say anything about the structure of that directory or why files are empty.

However, the purpose of a `dependency_links.txt` file would be to provide alternative package sources (such as the URL to a GitHub repository) for dependencies that might not be available on PyPi. They've long since been deprecated in favor of `requirements.txt` or `pyproject.toml`.